### DISCUSSION TITLE
Rendering Orchid Icon from TD

### DISCUSSION THREAD

**Discussion by Rfferrao87** [orchidsoftware/platform#2287] [Q&A] [answered]
I've been trying to use the `<x-orchid-icon>` component from a `Table` Layout class, but with no luck; I'd like to make sure this is correct and, if not so, what I should do to make it work please.

Here's a minimal example of what I'm trying to do in the `columns()` method (as in a table cell with an icon representing a _boolean_ value):

```
TD::make('active', __('Active'))
    ->render(function () {
        return '<x-orchid-icon path="check" />';
    })
```

**Comment by tabuna** [comment, marked as answer]
This will only output the text as `HTML`. If you want to pass a `Blade` you can do it like a `view()` example:

```php
TD::make('active', __('Active'))
    ->render(function () {
        return view('template');
    })
````

or use shorthand which will compile `Blade` syntax to `HTML`:

```php
use Illuminate\Support\Facades\Blade;
 
TD::make('active', __('Active'))
    ->render(function () {
        return Blade::render('<x-orchid-icon path="check" />');
    })
```
